plugins/obs-webrtc/webrtc/src/whip.rs

```diff
@@ -0,0 +1,84 @@
+use anyhow::Result;
+use log::{info, warn};
+use reqwest::{
+    header::{HeaderValue, AUTHORIZATION, CONTENT_TYPE, LOCATION},
+    Url,
+};
+use webrtc::peer_connection::sdp::session_description::RTCSessionDescription;
+
+use crate::obs_log;
+
+pub async fn offer(
+    url: &str,
+    bearer_token: Option<&str>,
+    local_desc: RTCSessionDescription,
+) -> Result<(RTCSessionDescription, Url)> {
+    let client = reqwest::Client::new();
+
+    let mut headers = reqwest::header::HeaderMap::new();
+    headers.insert(CONTENT_TYPE, HeaderValue::from_static("application/sdp"));
```

Comment:
Just a minor thing, I think it would be useful to also include the `User-Agent` header for both `POST` and `DELETE` WHIP queries.

I'm not sure what would be the ideal value to use, as the OBS codebase doesn't seem to be consistent when setting the user agent:
 * `obs-basic 1.2.3` in [remote-text](https://github.com/obsproject/obs-studio/blob/9140c260ee8ab9a11cf5716a1e3218bdb4873099/UI/remote-text.cpp#L42-L43)
 * `libobs/1.2.3` in [ffmpeg/HLS](https://github.com/obsproject/obs-studio/blob/cccfb0708055c2fb5b5873fa6ee90f71bf833941/plugins/obs-ffmpeg/obs-ffmpeg-hls-mux.c#L136-L137)

`libobs/${OBS_VERSION}` seems more "standard" to me.

Reply:
For our FLV mux, we use the string `obs-outputs (libobs version %d.%d.%d)`.

`libobs/%d.%d.%d` sounds like a good format to me.

Reply:
Fixed in b2827cd160ef8dfbf924de588d56b2f8d5ed48cc